apps/etl-func/src/adapters/pdv-tokenizer/problem.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from "zod";
+
+// Define the InvalidParam schema
+const invalidParamSchema = z.object({
+  name: z.string(),
+  reason: z.string(),
+});
+
+// Define the Problem schema
+export const problemSchema = z.object({
+  detail: z.string().optional(),
+  instance: z.string().optional(),
+  invalidParams: z.array(invalidParamSchema).optional(),
+  status: z.number().int(),
+  title: z.string(),
+  type: z.string().optional(),
+});
```

Comment:
are we using all these fields?

Reply:
This is the error response of the pdv tokenizer api. I don't know if we will use all of it but I mapped all of them in case the caller of the tokenizer would need it.